_quarto.yml

```diff
@@ -40,6 +40,9 @@ format:
       right: |
         This book was built with <a href="https://quarto.org/">Quarto</a>.
   pdf:
+    template-partials: 
+      - latex/title.tex
```

Comment:
To explain the changes in this PR; I needed to override some of the Quarto stuff to be able to make the callouts grey.  They add the colour definitions *after* the contents of "include-in-header", so I followed the advice in [this blog post](https://enrico.spinielli.net/posts/2023-05-01-quarto-callout-hack/) where we instead copy Quarto's default title.tex file and then add our colour definitions at the end of there.